changelogs/fragments/3500-macports-add-stdout-and-stderr-to-status.yaml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - macports - add ``stdout`` and ``stderr`` to ``status``
```

This is usually called 'return values' (see for example https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html):
```suggestion
  - macports - add ``stdout`` and ``stderr`` to return values
```